ospfd/ospf_vty.c

```diff
@@ -1566,6 +1566,58 @@ DEFUN (ospf_area_nssa,
 	return ospf_area_nssa_cmd_handler(vty, argc, argv, 0, 0);
 }
 
+DEFUN(ospf_area_nssa_suppress_fa, ospf_area_nssa_suppress_fa_cmd,
```

Comment:
Documentation and a topotest please to show that this works as well as keeping this working in the future

Reply:
okay, just pushed my first topotest...